app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/ProjectStoreImplicits.scala

```diff
@@ -33,6 +34,37 @@ import java.util.UUID
 
 class ProjectStoreImplicits(xa: Transactor[IO], mtr: MetricsRegistrator)
     extends ToProjectStoreOps {
+  implicit val sceneStore: ProjectStore[SceneDao] = new ProjectStore[SceneDao] {
+    def read(
+        self: SceneDao,
+        projId: UUID, // actually a scene id, but argument names have to match
+        window: Option[Projected[Polygon]],
+        bandOverride: Option[BandOverride],
+        imageSubset: Option[NEL[UUID]]): fs2.Stream[IO, BacksplashImage] = {
+      SceneDao.streamSceneById(projId).transact(xa) map { scene =>
+        BacksplashImage(
```

Comment:
Stylistically, I find it much harder to parse this than having the arguments split out:

```suggestion
       // we don't actually have a project, so just make something up
        val randomProjectId = UUID.randomUUID

        val ingestLocation = scene.ingestLocation getOrElse {
          throw UningestedScenesException(
            s"Scene ${scene.id} does not have an ingest location")
        }

        val footprint = scene.dataFootprint getOrElse {
          throw MetadataException(
            s"Scene ${scene.id} does not have a footprint"
          )
        }

        val imageBandOverride = bandOverride map { ovr =>
          List(ovr.red, ovr.green, ovr.blue)
        } getOrElse { List(0, 1, 2) }

        val colorCorrectParams = ColorCorrect.paramsFromBandSpecOnly(0, 1, 2)

        BacksplashImage(
          scene.id,
          randomProjectId,
          ingestLocation,
          footprint,
          imageBandOverride,
          colorCorrectParams,
          None // no single band options ever
        )
```
I don't think I can do a multiline suggested change - but this changest should work

Reply:
I knew you would hate this but forgot that I had done it between when I wrote it and after I'd tested it / opened the PR